src/ruleset.rs

```diff
@@ -719,6 +745,29 @@ fn ruleset_created_attr() {
     );
 }
 
+#[test]
+fn ruleset_created_new_ref() {
+    Ruleset::from(ABI::V1)
```

Comment:
Tests creating real rulesets needs to use the `check_ruleset_support()` helper and be in `lib.rs`. This is why running the test with Linux 5.10 failed (because Landlock is not supported with this version).